src/libicalvcal/vcc.c

```diff
@@ -216,9 +216,7 @@ DFARS 252.227-7013 or 48 CFR 52.227-19, as applicable.
 #include <config.h>
 #endif
 
-#include <string.h>
 #include <stdio.h>
-#include <stdlib.h>
```

Comment:
I *think* removing the #include <string.h> and #include <stdlib.h> starting at line 1403 is a better fix.  can you try that?

Reply:
No.  The lines are removed from vcc.y, and then yacc/bison generates vcc.c .  If I remove the lines 1403…1404, the change will appear again (moving #include <stdlib.h> from line 1403 to line 221).

Reply:
ok